mm2src/mm2_main/src/lp_swap/maker_swap.rs

```diff
@@ -1362,45 +1399,49 @@ impl MakerSwap {
             ),
             Err(e) => ERR!("Error {} when trying to find maker payment spend", e),
             Ok(None) => {
-                let can_refund_htlc = try_s!(
-                    self.maker_coin
-                        .can_refund_htlc(maker_payment_lock as u64)
-                        .compat()
-                        .await
-                );
-                if let CanRefundHtlc::HaveToWait(seconds_to_wait) = can_refund_htlc {
-                    return ERR!("Too early to refund, wait until {}", now_ms() / 1000 + seconds_to_wait);
-                }
-                let fut = self.maker_coin.send_maker_refunds_payment(SendMakerRefundsPaymentArgs {
-                    payment_tx: &maker_payment,
-                    time_lock: maker_payment_lock,
-                    other_pubkey: other_maker_coin_htlc_pub.as_slice(),
-                    secret_hash: secret_hash.as_slice(),
-                    swap_contract_address: &maker_coin_swap_contract_address,
-                    swap_unique_data: &unique_data,
-                });
-
-                let transaction = match fut.compat().await {
-                    Ok(t) => t,
-                    Err(err) => {
-                        if let Some(tx) = err.get_tx() {
-                            broadcast_p2p_tx_msg(
-                                &self.ctx,
-                                tx_helper_topic(self.maker_coin.ticker()),
-                                &tx,
-                                &self.p2p_privkey,
-                            );
-                        }
+                if self.taker_coin.is_auto_refundable() {
+                    ERR!("Taker payment will be refunded automatically!")
+                } else {
+                    let can_refund_htlc = try_s!(
+                        self.maker_coin
+                            .can_refund_htlc(maker_payment_lock as u64)
+                            .compat()
```

Comment:
Could you please refactor this as
```rust
if self.taker_coin.is_auto_refundable() {
    return ERR!("Taker payment will be refunded automatically!");
}

let can_refund_htlc = try_s!(
    self.maker_coin
        .can_refund_htlc(maker_payment_lock as u64)
        .compat()
        .await
);
```
This will allow to avoid whitespace changes :)